src/components/StockTable/styles.css

```diff
@@ -0,0 +1,13 @@
+table {
```

Comment:
I do not see these html tags being used anywhere in the code

Reply:
It's for the Chakra UI Table component since it uses regular html table code underneath. I was testing with the css a bit.